src/Gui/Forms/InitialPageInteractor.cs

```diff
@@ -182,5 +192,78 @@ private bool NeedsScanning()
                 Decompiler.Project != null &&
                 Decompiler.Project.Programs.Any(p => p.NeedsScanning);
         }
+
+        private void ProcessResources(ProgramResourceGroup prg, string outputDir)
+        {
+            try
+            {
+                if (prg.Name == "PE resources")
+                {
+                    // Create the dir
+                    Directory.CreateDirectory(outputDir + "\\resources");
+                    foreach (ProgramResourceGroup pr in prg.Resources)
+                    {
+                        switch (pr.Name)
+                        {
+                            case "CURSOR":
+                            {
+                                WriteResourceFile(outputDir + "\\resources", "Cursor", ".cur", pr);
+                            }
+                            break;
+                            case "BITMAP":
+                            {
+                                WriteResourceFile(outputDir + "\\resources", "Bitmap", ".bmp", pr);
+                            }
+                            break;
+                            case "ICON":
+                            {
+                                WriteResourceFile(outputDir + "\\resources", "Icon", ".ico", pr);
+                            }
+                            break;
+                            case "FONT":
+                            {
+                                WriteResourceFile(outputDir + "\\resources", "Font", ".bin", pr);
+                            }
+                            break;
+                            case "NEWBITMAP":
+                            {
+                                WriteResourceFile(outputDir + "\\resources", "NewBitmap", ".bmp", pr);
+                            }
+                            break;
+
+                            default:
+                                break;
+                        }
+                    }
+                }
+            }
+            catch (Exception)
+            {
```

Comment:
If the user runs out of file space or there is a permission error, they should be notified. Instead of discarding the exception, use:
```C#
catch (Exception ex) {
    var uiSvc = Services.RequireService<IDecompilerShellUiService>();
    uiSvc.ShowError(ex"A resource file could not be written.");
}
```